test/integration/serverless/run.sh

```diff
@@ -104,7 +112,14 @@ function remove_stack() {
 trap remove_stack EXIT
 
 # deploy the stack
-serverless deploy --stage "${stage}"
+if [ "$ARCHITECTURE" == "amd64" ]; then
+    # we use the architecture name amd64, while AWS requires it to be listed as x86_64
+    export DEPLOYMENT_ARCHITECTURE=x86_64
+    serverless deploy --stage "${stage}"
```

Comment:
Nit: It looks like this line is duplicated in both paths of the `if` statement - maybe there is a way to factor it out?

Reply:
fixed